vignettes/flowmaps-interactive.qmd

```diff
@@ -0,0 +1,381 @@
+---
+title: "Making interactive flow maps"
+vignette: >
+  %\VignetteIndexEntry{Making interactive flow maps}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+bibliography: references.bib
+number-sections: true
+format:
+  html:
+    toc: true
+    toc-depth: 2
+    code-overflow: wrap
+execute: 
+  eval: false
+---
+
+
+This tutorial shows how to make interactive 'flow maps' with data from `{spanishoddata}` and the `{flowmapblue}` [@flowmapblue_r] data visualisation package. We cover two examples. [First](#simple-example), we only visualise the total flows for a single day. In the [second](#advanced-example) more advanced example we also use the time component that allows you to interactively filter flows by time of day. For both examples, make sure you first go though the initial [setup steps](#setup). To make static flow maps, please see the [static flow maps](https://ropenspain.github.io/spanishoddata/articles/flowmaps-static.html) tutorial.
+
+
+# Setup {#setup}
+
+For the basemap in the final visualisation you will need a free Mapbox access token. You can get one at [account.mapbox.com/access-tokens/](https://account.mapbox.com/access-tokens/){target='_blank'} (you need to have a Mapbox account, which is free). You may skip this step, but in this case your interative flowmap will have no basemap, and the flows will just flow on solid colour background.
+
+Once you got the access token, you can set it in the `MAPBOX_TOKEN` environment variable like so:
+
+```{r}
+#| eval: false
+Sys.setenv(MAPBOX_TOKEN = "YOUR_MAPBOX_ACCESS_TOKEN")
+```
+
+
+```{r}
+library(spanishoddata)
+library(flowmapblue)
+library(tidyverse)
+library(sf)
+```
+
+{{< include ../inst/vignette-include/setup-data-directory.qmd >}}
```

Comment:
So handy!

Reply:
indeed! it took a while to figure out where to put the qmd to make both pkgdown and r cmd check happy